src/assets/properties-panel.css

```diff
@@ -299,6 +301,10 @@
   background: none;
 }
 
+.bio-properties-panel-group-header-button:focus {
+  outline-offset: 2px;
```

Comment:
Outline still black for me:

<img width="237" alt="brave_gxcOzPHfJA" src="https://github.com/user-attachments/assets/ed4af008-b254-47cc-9040-a3fc3abf10bb">

Suggestion:

```suggestion
  outline-offset: 2px;
  outline-color: var(--focus-outline-color);
```

Reply:
That's strange, it's already showing blue for me. Are you linking the library to `bpmn-js-properties-panel` to verify the changes?

Even before my updates, the outline was already blue; I only added the offset. Could you double-check? I want to avoid adding redundant properties.

Reply:
Just checked again and it's still black. Where would the blue color come from?

Reply:
It was coming from `-webkit-focus-ring-color` (default focus color). I have updated the code and added the outline color property.